WowPacketParserModule.V7_0_3_22248/Parsers/GarrisonHandler.cs

```diff
@@ -16,24 +16,16 @@ public static void ReadGarrisonMission(Packet packet, params object[] indexes)
             packet.ReadTime("StartTime", indexes);
             packet.ReadUInt32("TravelDuration", indexes);
             packet.ReadUInt32("MissionDuration", indexes);
-            packet.ReadUInt32("MissionState", indexes);
-            packet.ReadUInt32("Unknown1", indexes);
-            packet.ReadUInt32("Unknown2", indexes);
+            packet.ReadUInt32E<GarrisonMissionState>("MissionState", indexes);
+            packet.ReadUInt32("SuccessChance", indexes);
+            packet.ReadUInt32E<GarrisonMissionFlag>("Flags", indexes);
```

Comment:
are u sure about this field?

Reply:
Companion receives same packets and handels this part as flag.. so yeah pretty